modules/ROOT/pages/sap/sap-s-4hana-cloud-connector-reference.adoc

```diff
@@ -0,0 +1,1082 @@
+| Buffer Size Increment a| Number | This specifies how much the buffer size expands if it exceeds its initial size. Setting a value of zero or lower means that the buffer should not expand, meaning that a STREAM_MAXIMUM_SIZE_EXCEEDED error is raised when the buffer gets full. |  | 
+| Max Buffer Size a| Number | The maximum amount of memory to use. If more than that is used then a STREAM_MAXIMUM_SIZE_EXCEEDED error is raised. A value lower than or equal to zero means no limit. |  | 
+| Buffer Unit a| Enumeration, one of:
+
+** BYTE
+** KB
+** MB
+** GB | The unit in which all these attributes are expressed |  | 
+|===
+
+[[RepeatableFileStoreStream]]
+=== Repeatable File Store Stream
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| In Memory Size a| Number | Defines the maximum memory that the stream should use to keep data in memory. If more than that is consumed then it will start to buffer the content on disk. |  | 
+| Buffer Unit a| Enumeration, one of:
+
+** BYTE
+** KB
+** MB
+** GB | The unit in which maxInMemorySize is expressed |  | 
+|===
+
+[[ResponseAttributes]]
+=== Response Attributes
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Headers a| Object |  |  | 
+| Status Code a| Number |  |  | 
+|===
+
+[[BulkOperationResult]]
+=== Bulk Operation Result
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Id a| Any |  |  | 
+| Items a| Array of <<BulkItem>> |  |  | 
+| Successful a| Boolean |  |  | 
+|===
+
+[[BulkItem]]
+=== Bulk Item
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Exception a| Any |  |  | 
+| Id a| Any |  |  | 
+| Message a| String |  |  | 
+| Payload a| Any |  |  | 
+| Status Code a| String |  |  | 
+| Successful a| Boolean |  |  | 
+|===
+
+[[RepeatableInMemoryIterable]]
+=== Repeatable In Memory Iterable
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Initial Buffer Size a| Number | The amount of instances that is initially be allowed to be kept in memory to consume the stream and provide random access to it. If the stream contains more data than can fit into this buffer, then the buffer expands according to the bufferSizeIncrement attribute, with an upper limit of maxInMemorySize. Default value is 100 instances. |  | 
+| Buffer Size Increment a| Number | This is by how much the buffer size expands if it exceeds its initial size. Setting a value of zero or lower means that the buffer should not expand, meaning that a STREAM_MAXIMUM_SIZE_EXCEEDED error is raised when the buffer gets full. Default value is 100 instances. |  | 
```

Comment:
The default value should be in the Default Value column,

Reply:
Changed.